src/shared-styles/_logical-properties.scss

```diff
@@ -1 +1,290 @@
+@use "direction";
 @use "rem";
+@use "validation";
+
+@function _maybe-rem($to-rem, $values) {
+  @if ($to-rem == false) {
+    @return $values;
+  } @else {
+    @return rem.calculate($values...);
+  }
+}
+
+@mixin _maybe-rem($to-rem, $properties, $values: ()) {
+  @if ($to-rem == false) {
+    @if type-of($properties) == "map" {
+      @each $property in map-keys($properties) {
+        @include _maybe-rem($to-rem, $property, map-get($properties, $property));
+      }
+    } @else {
+      @each $property in $properties {
+        #{$property}: $values;
+      }
+    }
+  } @else {
+    @include rem.calculate($properties, $values...);
+  }
+}
```

Comment:
Since we don't need rem fallbacks anymore, I'd prefer to just use the function and declare properties normally.

Reply:
I'll look at how things could be simplified.